ludwig/api.py

```diff
@@ -546,6 +547,45 @@ def train(
 
         return train_stats
 
+    def kfold_cross_validate(
+            self,
+            k_fold,
+            data_csv=None,
```

Comment:
It should accept all formats of data that `train` adn `predict` accept

Reply:
updated function signatures